relval_submit.py

```diff
@@ -18,6 +18,8 @@
 import json
 import errno
 import ast
+sys.path.insert(0, './modules')
```

Comment:
how do you launch relval_submit that you need sys path insert? The normal: from modules import wma should work, but i see that you made an explicit commit for this change.